src/logic/BrazeCards.test.ts

```diff
@@ -0,0 +1,247 @@
+import fc from 'fast-check';
+import appboy from '@braze/web-sdk-core';
+import { BrazeCard, BrazeCards } from './BrazeCards';
+import { CardSlotName, Extras } from './types';
+
+function errorHandler(error: Error, identifier: string): void {
+    console.log(identifier, error);
+}
+
+describe('braze content cards', () => {
+    describe('BrazeCards', () => {
+        describe('getCardsForProfileBadge', () => {
+            test('returns empty array if there are no content cards for this user', () => {
```

Comment:
```suggestion
            it('returns empty array if there are no content cards for this user', () => {
```

Super minor and nit-picky, but we've tended to use `it` for these assertion blocks elsewhere in the codebase, if we want to be consistent here (I also like the way the entire line reads with `it`).